s2i-nodejs/s2i-nodejs-task.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: s2i-nodejs
+spec:
+  inputs:
+    resources:
+      - name: source
+        type: git
+    params:
+      - name: VERSION
+        description: The version of the nodejs
+        default: 10
+      - name: PATH_CONTEXT
+        description: The location of the path to run s2i from.
+        default: .
+      - name: TLSVERIFY
+        description: Verify the TLS on the registry endpoint (for push/pull to a non-TLS registry)
+        default: "true"
+  outputs:
+    resources:
+      - name: image
+        type: image
+  steps:
+    - name: generate
+      image: quay.io/openshift-pipeline/s2i
+      workingdir: /workspace/source
+      command: ['s2i', 'build', '${inputs.params.PATH_CONTEXT}', 'centos/nodejs-${inputs.params.VERSION}-centos7', '--as-dockerfile', '/gen-source/Dockerfile.gen']
```

Comment:
Did you talk to the RHOAR Node.js team about using their image(s)?

Reply:
The Node.js images are used here like any other S2I use-cases. Are there any concerns?

Reply:
I don't have any concerns about how the images are used. I'm just wondering about _which_ images are used. I see that for Java, the images produced by the Red Hat OpenJDK team are used. Why not use the Node.js images produced by the RHOAR Node.js team? I believe they also have a community, CentOS-based variants, if that's a concern.

Reply:
Sure. That's the goal. It will be added #4